CHANGELOG.md

```diff
@@ -11,6 +11,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - removed bip39 and mnemonic feature ([#86])
 
+## [0.4.0] - 2019-05-20
+
+### Changed
+
+- updated vendorField to support 255 bytes in Core v2.4 ([#84])
```

Comment:
When you add those numbers in the brackets you also need to add them here https://github.com/ArkEcosystem/cpp-crypto/pull/86/files/90bea179a4eb262800b98fad60064d0a616132a2..dc7034f4fbafb49c828996cbc2031cc24bdcead9#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR92.